R/module.R

```diff
@@ -77,3 +79,17 @@ odin_ui_get_time <- function(x, has_start_time) {
 odin_ui_get_output <- function(x, map) {
   vlapply(map, function(el) x[[el]])
 }
+
+
+odin_ui_get_graph_options <- function(input, name_map) {
+  include <- odin_ui_get_output(input, name_map)
+  palette <- get(input$choice_palette)
+  colors <- set_names(palette(length(include)), names(include))
+  list(include = include,
+       colors = colors,
```

Comment:
can we use `cols` rather than getting involved in the `colors`/`colours` bunfight (with `colors` being *obviously* incorrect :upside_down_face: )

Reply:
actually cols confused me the first time, I wasn't sure if it referred to colors or columns.. 

but if you want I'll settle for 'couleur'? ;)